database/PostgreSQL/database.js

```diff
@@ -1,18 +1,56 @@
-const { Sequelize } = require('sequelize');
+const { Pool } = require('pg');
+const pgtools = require('pgtools');
+const PASSWORD = require('../passwordIgnore.js');
 
-const sequelize = new Sequelize('images', 'postgres', 'Wantagh1!@#', {
+const config = {
+  user: 'postgres',
   host: 'localhost',
-  dialect: 'postgres'
+  password: PASSWORD.PASSWORD,
+  port: 5432,
+}
+
+const pool = new Pool({
+  user: 'postgres',
+  host: 'localhost',
+  database: 'images',
+  password: PASSWORD.PASSWORD,
+  port: 5432
 });
```

Comment:
This seems redundant to me, but I haven't looked into the docs for the pg module yet. Do you need to include the user, host, password, and port in both places?